src/gui/EffectSelectDialog.cpp

```diff
@@ -31,6 +31,7 @@
 #include "PluginFactory.h"
 
 #include <QLabel>
+#include <QHeaderView>
```

Comment:
Please drop this, it is already fixed in  #4374 and synced into `master`.

Reply:
No problem. I was sure I was the cause somehow (since it just appeared from no where)